projects/core/src/cms/page/base-page-meta.resolver.ts

```diff
@@ -80,7 +80,7 @@ export class BasePageMetaResolver
     options?: CanonicalUrlOptions,
     url?: string
   ): Observable<string> {
-    return of(this.buildCanonicalUrl(options, url));
+    return this.page$.pipe(map(() => this.buildCanonicalUrl(options, url)));
```

Comment:
Storefinder has one CMS page, but many child routes. I think we shouldn't update cannonical URL on the change of CMS page, but rather on any change in the route.
One might say "in SSR it will work anyway, let's not bother". But since we provide flag `enableInDevMode` to simulate the cannonical URL in realtime, I think we should make it bullet proof.

Reply:
Good point.